### PR TITLE
Fix resources in containers

### PR DESCRIPTION
**Pull request by jansupol**
Signed-off-by: Jan Supol <jan.supol@oracle.com>

### CODE REVIEW

File: containers/grizzly2-http/pom.xml [hunk 1]
```diff
@@ -61,19 +61,6 @@
                 <inherited>true</inherited>
             </plugin>
         </plugins>
-
-        <resources>
-            <resource>
-                <directory>${basedir}/src/main/java</directory>
```
Comment (Tomas-Kraus) [inline containers/grizzly2-http/pom.xml:67]: Why do you have META-INF in src/main/java? I would move it to src/main/resources.

File: containers/jetty-http/pom.xml [hunk 2]
```diff
@@ -73,29 +73,11 @@
...
             <resource>
                 <directory>${basedir}/src/main/resources</directory>
                 <filtering>true</filtering>
             </resource>
         </resources>
     </build>
 
-    <profiles>
-        <profile>
-            <id>testsSkipJdk6</id>
```
Comment (Tomas-Kraus) [inline containers/jetty-http/pom.xml:91]: Does it make sense to support such historical artifact?

Comment (senivam) [reply containers/jetty-http/pom.xml:91]: probably that is why this profile is being removed...

File: containers/grizzly2-http/pom.xml (hunk 1 above)
Comment (senivam) [reply containers/grizzly2-http/pom.xml:67]: this whole block is redundant and is being removed from Jersey...

File: containers/jetty-http/pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply containers/jetty-http/pom.xml:91]: Yes. My fault, I somehow did not realize that this is "red". :)

File: containers/grizzly2-http/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply containers/grizzly2-http/pom.xml:67]: Yes, but still, things like META-INF shall be in `src/main/resources`.

**Pull request merged**